src/java/winstone/AbstractSecuredConnectorFactory.java

```diff
@@ -0,0 +1,233 @@
+/*
+ * Copyright Olivier Lamy
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Yes, you can. Maybe makes sense to add a comment just in case somebody runs license verification tools which raise a warning